app/helpers/quadicon_helper.rb

```diff
@@ -216,6 +217,48 @@ def render_quadicon_label(item, row)
     end
   end
 
+  # Renders a quadicon for PhysicalServer
+  #
+  def render_physical_server_quadicon(item, options)
+    output = []
+    if settings(:quadicons, :physical_server)
+      output << flobj_img_simple("layout/base.png")
+
+      output << flobj_p_simple("a72", (item.host ? 1 : 0))
+      output << flobj_img_simple("svg/currentstate-#{h(item.power_state.downcase)}.svg", "b72")
+      output << flobj_img_simple(item.ext_management_system.decorate.fileicon, "c72")
+      output << flobj_img_simple(img_for_health_state(item), "d72")
+      output << flobj_img_simple('100/shield.png', "g72") unless item.get_policies.empty?
+    else
+      output << flobj_img_simple(size)
+      output << flobj_img_simple(width * 1.8, item.ext_management_system.decorate.fileicon, "e72")
```

Comment:
And the first arg here is dead too.. `output << flobj_img_simple(item.ext_management_system.decorate.fileicon, "e72")` should do the trick.